front/components/PlansTables.tsx

```diff
@@ -0,0 +1,286 @@
+      <PriceTable.Item label="Unlimited custom assistants" />
+      <PriceTable.Item label="Unlimited messages" />
+      <PriceTable.Item label="Up to 1Go/user of data sources" />
+      <PriceTable.Item
+        label="Connections
+  (GitHub, Google Drive, Notion, Slack)"
+      />
+      <PriceTable.Item label="Single Sign-on (Google, GitHub)" />
+      <PriceTable.Item label="Dust Slackbot" />
+      <PriceTable.Item label="Assistants can execute actions" />
+      <PriceTable.Item label="Workspace role and permissions" variant="dash" />
+      <PriceTable.ActionContainer>
+        {plan.code !== "PRO_PLAN_SEAT_29" && (
+          <Button
+            variant="primary"
+            size={biggerButtonSize}
+            label="Start now"
+            icon={RocketIcon}
+            disabled={isProcessing || plan.code === "PRO_PLAN_SEAT_29"}
+            onClick={onClick}
+          />
+        )}
+      </PriceTable.ActionContainer>
+    </PriceTable>
+  );
+}
+function EnterprisePriceTable({
+  size,
+  onClick,
+  isProcessing,
+}: {
+  size: "sm" | "xs";
+  onClick: () => void;
+  isProcessing: boolean;
+}) {
+  const biggerButtonSize = size === "xs" ? "sm" : "md";
+  return (
+    <PriceTable title="Enterprise" price="Custom" size={size} magnified={false}>
+      <PriceTable.Item label="From 100 users" />
+      <PriceTable.Item label="Multiple workspaces" />
+      <PriceTable.Item label="Privacy and Data Security" />
+      <PriceTable.Item label="Advanced LLM models (GPT-4, Claude…)" />
+      <PriceTable.Item label="Unlimited custom assistants" />
+      <PriceTable.Item label="Unlimited messages" />
+      <PriceTable.Item label="Unlimited data sources" />
+      <PriceTable.Item
+        label="Connections
+  (GitHub, Google Drive, Notion, Slack…)"
+      />
+      <PriceTable.Item label="Single Sign-on" />
+      <PriceTable.Item label="Dust Slackbot" />
+      <PriceTable.Item label="Assistants can execute actions" />
+      <PriceTable.Item label="Advanced workspace role and permissions" />
+      <PriceTable.Item label="Dedicated account support" />
+      <PriceTable.ActionContainer>
+        <Button
+          variant="primary"
+          size={biggerButtonSize}
+          label="Contact us"
+          icon={SparklesIcon}
+          disabled={isProcessing}
+          onClick={onClick}
+        />
+      </PriceTable.ActionContainer>
+    </PriceTable>
+  );
+}
+
+export function PricePlans({
+  size = "sm",
+  isTabs = false,
+  className = "",
+  plan,
+  onClickProPlan,
+  onClickTestPlan,
+  onClickEnterprisePlan,
+  isProcessing,
+}: PricePlanProps) {
+  if (isTabs) {
+    return (
+      <div
+        className={classNames(
+          "mx-0 sm:mx-24",
+          "w-full max-w-md px-2 py-16 sm:px-0",
+          className
+        )}
+      >
+        <Tab.Group>
+          <Tab.List className="flex space-x-1 rounded-full border border-slate-600/40 bg-slate-900/40 p-1 backdrop-blur">
```

Comment:
Need to create a "light" version of this.
You can copy the style of the BarHeader.